consul/src/main/scala/io/buoyant/consul/v1/KvApi.scala

```diff
@@ -95,15 +121,27 @@ class KvApi(
     path: String,
     datacenter: Option[String] = None,
     cas: Option[String] = None,
+    recurse: Boolean = false,
     retry: Boolean = false
```

Comment:
The same situation here as well - separator is accepted only for `get` method and only when `list` query parameter is used.